### DISCUSSION TITLE
This terminal is awesome!

### DISCUSSION THREAD

**Discussion by jrmoulton** [wezterm/wezterm#2752] [General]
Hopefully this doesn't add too much noise but I just wanted to drop a comment saying that this terminal is awesome! And thanks for developing and maintaining it. 

I used to use Alacritty and then switched to Kitty for ligatures but have definitely noticed how slow kitty can be. I've recently switched to wezterm and it was super easy to configure. Very straightforward and it works and looks great! And it's fast! And it has ligatures!

This is the perfect combo for me and I just wanted to drop a thank you

**Comment by wez** [comment]
Thanks for the kind words!

**Comment by fabschurt** [comment]
Hi @wez, allow me to reuse this thread and state that I also **love** WezTerm.

It’s been a long time that I’ve been looking for a terminal emulator that would neatly fit my needs, and it’s not easy because I spend a lot of time on the command line and it makes me picky, but this is it! WezTerm hits the very sweet spot between simplicity and power/extensibility. (Lua was a great choice!)

Thank you very much for all this hard work, it is truly appreciated :heart:

**Comment by askegg** [comment]
+1

I've been looking at various terminal options lately, and Wezterm is exactly what I was looking for. Thank you 🙌
